content/docs/connectors.md

```diff
@@ -0,0 +1,634 @@
+---
+draft: false
+title: "Syndesis connectors development"
+sidebar: "sidenav"
+menu:
+  sidenav:
+    name: Connectors development
+    weight: 3
+    parent: Developer Docs
+toc: true
+weight: 20
+---
+
+One of the most important features of Syndesis is to be able to connect to a variety of sources and destinations. The middleware developed is going to take care of transparently of the integration to such source/destination of data requiring a custom development of the specific transport involved in the chosen component.
+
+Since Apache Camel is providing a large [offering of integrations components](https://camel.apache.org/components.html), Syndesis does offer the possibility to use them thanks to a proxy component whose goal is to identify the component and provide the correct configuration settings to make them available through the user interface.
+
+### Tutorial
+We will use as a reference the development of a connector for MongoDB and we will try to show explicitly all the configuration and development you will have to perform in order to develop your own connector.
+
+If you haven’t done yet, fork the syndesis project, check it out locally and familiarize with its structure. Also you may want to read the [engineering guidelines](https://syndesis.io/community/contributing/#1-fork-the-repository) before starting to work on it.
+### Camel upstream component
+The first thing you will have to do is to have a look at the list of camel components provided in the introduction. Fortunately there is quite a lot of components available, so you should be lucky as in the example I’m going to show for [MongoDB](https://camel.apache.org/mongodb.html).
```

Comment:
"you should be lucky" or "you can be lucky"?

Reply:
No, I meant should, as probably there will be some component for the connector that has to be developed.